### PR TITLE
Added wishlist page

### PR DESCRIPTION
**Pull request by nishasen**
In this page products which the user add to wishlist are shown, it also has links to go back to product listing page.

image

### CODE REVIEW

**Review by sadathshariff — APPROVED** [review]
Good Naming conventions and the code looks clean

**Review by anushasunil — APPROVED** [review]
Code looks well structured and easy to understand. Liked the use of Toasts. Great work ✨

File: diagon-alley/src/Components/WishCartHeader/WishCartHeader.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.wish-cart-header {
+    font-size: 1.5rem;
+    font-weight: bold;
+    margin: 2rem auto;
+    margin-top: 5rem;
+    text-align: center;
+}
+
+.header-span {
+    background-color: hsla(0, 0%, 98%, 0.8);
```
Comment (anushasunil) [inline diagon-alley/src/Components/WishCartHeader/WishCartHeader.css:10]: can use variables instead

File: diagon-alley/src/Components/WishlistCard/WishlistCard.module.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.product_card {
+    background-color: hsla(0, 0%, 98%, 0.7);
```
Comment (anushasunil) [inline diagon-alley/src/Components/WishlistCard/WishlistCard.module.css:2]: here also

**Review by anushasunil — COMMENTED** [review]
just small suggestions

File: diagon-alley/src/Components/WishCartHeader/WishCartHeader.css (hunk 1 above)
Comment (nishasen) [reply diagon-alley/src/Components/WishCartHeader/WishCartHeader.css:10]: Actually it is different in different places so added it this way here

**Pull request merged**